tests/util/kubeUtils.go

```diff
@@ -225,15 +225,20 @@ func GetPodsName(n string) (pods []string) {
 
 // GetPodStatus gets status of a pod from a namespace
```

Comment:
Could you include your findings and explanation to the function doc? Thanks!

Reply:
Done